tests/app.js

```diff
@@ -42,6 +42,10 @@ jet.exposeContextProperty('NativeModules', NativeModules);
 jet.exposeContextProperty('NativeEventEmitter', NativeEventEmitter);
 jet.exposeContextProperty('module', firebase);
 
+const db = firestore();
+
+db.settings({ host: 'localhost:8080', ssl: false, persistence: false });
```

Comment:
I'd set persistence to true, then all the cache related tests above that you've commented out should now work.

Reply:
ha, yep, that would make sense actually.